tests/integration/suite/daprd/binding/input/appready.go

```diff
@@ -99,6 +99,7 @@ spec:
 func (a *appready) Run(t *testing.T, ctx context.Context) {
 	a.daprd.WaitUntilRunning(t, ctx)
 
+	a.bindingCalled.Store(0)
```

Comment:
This is superfluous as its zeroth value is 0

Reply:
the test was erring due to that being 1 before it gets called later on in the test

Reply:
It should never be 1 because the app isn't healthy yet so the binding should not be called. If it is, there is a bug.

Reply:
maybe this is a flaky bug then. I can remove this change, but will keep it on the radar